packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -112,10 +112,10 @@ describe.each([['petstore.oas2.json'], ['petstore.oas3.json']])('server %s', fil
 
   it('should not mock a verb that is not defined on a path', async () => {
     const response = await server.fastify.inject({
-      method: 'POST',
```

Comment:
In case you're asking yourself: yes, this test has been wrong since forever. It's now fixed.